submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/submitter/LocalExtSubmissionSubmitter.kt

```diff
@@ -63,67 +65,143 @@ class LocalExtSubmissionSubmitter(
         return requestService.createRequest(request)
     }
 
+    @Suppress("CyclomaticComplexMethod")
+    override suspend fun completeRqt(
+        accNo: String,
+        version: Int,
+    ) {
+        suspend fun fromSavedSubmission() {
```

Comment:
Nice abstraction! 👍 

Could we move these to a different location? Maybe in the companion? Even though it clarifies what's going on and where the request is starting from, it was a bit hard to read at one glance. I had to read the code a couple of times to understand where the request was being processed as a whole. 

Having these methods in a separate place, would make it cleaner and clearer for someone trying to understand the flow starting from [here](https://github.com/EBIBioStudies/biostudies-backend-services/pull/884/files#diff-cce2eacbefada70c4273ebfc61a1bf521a71ed09054adb7287f259b748f2bbb8R119)

Reply:
they can not go in the companion as they use the requests handlers and companion only allow static, another option is to put them in the class itself but IMO they will add too much noise as this methods only are related to this operation. Any suggestion is welcome